frontend/src/app/chapters/[chapterKey]/page.tsx

```diff
@@ -60,15 +61,22 @@ export default function ChapterDetailsPage() {
     },
   ]
   return (
-    <DetailsCard
-      details={details}
-      geolocationData={chapter}
-      is_active={chapter.isActive}
-      socialLinks={chapter.relatedUrls}
-      summary={chapter.summary}
-      title={chapter.name}
-      topContributors={topContributors}
-      type="chapter"
-    />
+    <PageLayout
+      bcItems={[
```

Comment:
Same here - can we update this for a clearer naming? If it's conflicting with anything we can do something like `crumbItem`.